.github/workflows/swift-toolchain.yml

```diff
@@ -881,17 +877,19 @@ jobs:
           show-progress: false
 
       - name: Install Python ${{ env.PYTHON_VERSION }} (Host)
+        if: matrix.arch == 'amd64' || inputs.build_arch == 'amd64'
         uses: actions/setup-python@v5
         id: python
         with:
           python-version: '${{ env.PYTHON_VERSION }}'
+          architecture: x64
```

Comment:
Should this not be `${{ inputs.build_arch }}`?

Reply:
No, as it's used for `amd64` only